ts/components/wallet/paymentSummary/UpdatedPaymentSummaryComponent.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * This component will display the transaction details if updates on the amount are identified
+ */
+import { H1, H3, Icon, Right, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletStyles } from "../../../components/styles/wallet";
+import I18n from "../../../i18n";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+  amount: string;
+  updatedAmount: string;
+  expireDate: string;
+  tranche: string;
+}>;
+
+const styles = StyleSheet.create({
+  padded: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  },
+
+  whiteStrike: {
+    fontSize: variables.fontSizeBase * 1.25,
+    color: variables.brandLight,
+    textDecorationLine: "line-through",
+    textDecorationStyle: "solid"
+  },
+
+  toAlignCenter: {
+    flexDirection: "row",
+    alignItems: "baseline"
+  },
+
+  toAlignColumnstart: {
+    flexDirection: "column",
+    alignItems: "flex-start"
+  },
+
+  underlined: {
+    textDecorationLine: "underline",
+    color: variables.brandLight
+  },
+
+  /**
+   * TODO: testare proporzioni
+   */
+  iconMargin: {
+    marginBottom: 3,
+    marginLeft: 3
+  }
+});
+
+export default class UpdatedPaymentSummaryComponent extends React.Component<
```

Comment:
This looks an awful lot like `PaymentSummaryComponent`, with just a few minor changes based on whether the amount has been updated. You should merge the two components into a single one, and add the extra part if the amounts are different:
```
<Grid ...>
  { amounts_are_different && (
    <Row ... [row with new amount here ]>
  )}
  ...
```
^ This, by the way, leverages the fact that javascript (much like other languages) evaluates AND-concatenated boolean expression from left to right and stops when a definite value can be inferred. For an AND-concatenated expression to be true, all its parts need to be true, so if the first one is false (in this case, `amounts_are_different`, the engine won't even bother checking the rest of the expression, thus not evaluating the `Row` displaying the updated stuff. 

You may also need to use some ternary operator to make things easier when defining the original price's style: 
```
<H1 style={amounts_are_different ? styles.whiteStrike : WalletStyles.white}>...</H1>
```
More generally, the ternary operator has the syntax:
```
(condition) ? value_if_condition_is_true : value_if_condition_is_false
```